docs/appendix.md

```diff
@@ -1,30 +0,0 @@
-# Appendix
```

Comment:
💡Idea share: Let's do all the docs update in separate PR, there is history to various docs, and we can cater them all separately.

Reply:
I only left the new file docs/kustomize.md and reverted the others